backend/tenant_account_v2/enums.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class UserRole(Enum):
+    USER = "user"
```

Comment:
@muhammad-ali-e are these the roles configured at Auth0?

Reply:
@johnyrahul No. this enum for oss , We don't need to show any third parties roles here.